src/docs/reference/modules/Media.Amazon/README.md

```diff
@@ -0,0 +1,91 @@
+# Amazon Media Storage (`OrchardCore.Media.AmazonS3`)
+
+The Amazon Media Storage feature enables support for storing assets in Amazon S3 Bucket.
+
+The feature replaces the default App_Data file based media store with an Amazon Media Storage Provider.
+
+Media is still served by the Orchard Core web site, and the Media Cache module takes responsibility for fetching media, on the fly, from the Amazon S3 Bucket.
+
+This allows the Amazon Media Storage feature to support image resizing on the fly through integration with ImageSharp.Web.
+
+The url generated by the AssetUrl helpers, points to the Orchard Core web site.
+
+## Configuration
+
+The following configuration values are used by default and can be customized:
+
+```json
+{
+   "OrchardCore": {
+       "OrchardCore_Media_Amazon_S3": {
+           // Your AWS S3 Bucket name
+           "BucketName": "",
+           // This section needed only if Orchard will be hosted not in the AWS Cloud
+           // You can obtain all that information in the IAM Management Console
+            "Credentials": {
+              "SecretKey": "",
+              "AccessKeyId": "",
+              "RegionEndpoint": ""
```

Comment:
Please add some guidance to the docs on where to get this from. I tried the corresponding endpoints from https://docs.aws.amazon.com/general/latest/gr/s3.html, as well as using a more generic eu-central-1.amazonaws.com endpoint (for a Frankfurt-based bucket), but every single one of them cause "ArgumentException: Invalid region endpoint provided".

Reply:
I've added small comment about region in the doc file. Basically you need to use values from Region column, in your case just `eu-central-1`, but as I said I've added comment in doc.